workflows/checks/check-md5.nf

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.run_metafile = 's3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv'
+params.outdir = 's3://nextflow-ccdl-results/scpca-checks/'
+
+process check_md5{
```

Comment:
Ah so a process is like a custom function essentially.

Reply:
Yes, but a bit more. These are the steps that will run independently. So each process has to be able to run on a separate machine, and it defines all the files and resources it needs.

Reply:
Gotchaa!